test/diagnostics/verifier.swift

```diff
@@ -16,7 +16,7 @@ let y: Int = "hello, world!" // expected-error@:49 {{cannot convert value of typ
 // Wrong fix-it
 let z: Int = "hello, world!" as Any
 // expected-error@-1 {{cannot convert value of type}} {{3-3=foobarbaz}}
-// CHECK: expected fix-it not seen; actual fix-its: {{[{][{]}}36-36= as! Int{{[}][}]}}
+// CHECK: expected fix-it not seen; actual fix-it seen: {{[{][{]}}36-36= as! Int{{[}][}]}}
```

Comment:
Does this make sense to be `actual fix-its seen:` because it can have more than one? :)

Reply:
Thanks. I will support plural form.